modules/efk-logging-driver-viewing.adoc

```diff
@@ -7,7 +7,7 @@
 
 Fluentd uses the *journald* logging driver as the system log source. These are log messages from the operating system, the container runtime, and OpenShift.
 
-For EFK logging, you should used the *json-file* driver. See _Changing the Aggregated Logging Driver_ for instructions on changing from *journald* to *json-file*.
+For cluster logging, you should use the *json-file* driver. See _Changing the Aggregated Logging Driver_ for instructions on changing from *journald* to *json-file*.
```

Comment:
I'd try to remove the reference, but if you can't, I'd offset Changing the Aggregated Logging Driver with * instead of _. 
I'd also remove "should."